SSZipArchive/minizip/zip.c

```diff
@@ -627,6 +627,13 @@ extern zipFile ZEXPORT zipOpen4(const void *path, int append, uint64_t disk_size
     else
         ziinit.z_filefunc = *pzlib_filefunc64_32_def;
 
+    if (!(ziinit.z_filefunc.zopen32_file || ziinit.z_filefunc.zfile_func64.zopen64_file) ||
```

Comment:
This fix is perhaps not necessary, but it's good practice and silences 7 "FORWARD_NULL" issues. It ensures that at least one of each pair of function pointers is not NULL.